android/src/main/java/io/bluedot/BluedotPointSdkModule.java

```diff
@@ -69,28 +75,241 @@ private void sendEvent(ReactContext reactContext,
+                            serviceManager.subscribeForApplicationNotification(this);
+                            onSuccess.invoke("Success");
+                        });
+            }
+        } else {
+            //Start as No FG Service
+            GeoTriggeringService.builder()
+                    .start(reactContext, geoTriggerError -> {
+                        if (geoTriggerError != null) {
+                            onError.invoke("Error " + geoTriggerError.getReason());
+                            return;
+                        }
+                        serviceManager.subscribeForApplicationNotification(this);
+                        onSuccess.invoke("Success");
+                    });
+        }
+    }
+
+    @ReactMethod
+    public void isGeotriggeringRunning(Promise promise) {
+        try {
+            boolean isRunning = GeoTriggeringService.isRunning();
+            promise.resolve(isRunning);
+        } catch (Exception e) {
+            promise.reject("Error getting isGeotriggeringRunning");
+        }
+    }
+
+    @ReactMethod
+    public void stopGeotriggering(Callback onSuccessCallback, Callback onFailCallback){
+        GeoTriggeringStatusListener statusListener = error -> {
+            if (error == null) {
+                serviceManager.unsubscribeForApplicationNotification(this);
+                onSuccessCallback.invoke("Success");
+                return;
+            }
+            serviceManager.unsubscribeForApplicationNotification(this);
+            onFailCallback.invoke(error.getReason());
+        };
+        GeoTriggeringService.stop(reactContext, statusListener);
+    }
+
+    @ReactMethod
+    public void androidStartTempoTracking(String destinationId,
+                                          String channelId,
+                                          String channelName,
+                                          String androidNotificationTitle,
+                                          String androidNotificationContent,
+                                          Integer androidNotificationId,
+                                          Callback onSuccess,
+                                          Callback onError) {
+
+        if (destinationId == null) {
+            onError.invoke("destinationId is null");
+            return;
+        }
+
+        if ((channelId == null) || (channelName == null) || (androidNotificationTitle == null) || (androidNotificationContent == null)) {
+            onError.invoke("Missing param from channelId/channelName/androidNotificationTitle/androidNotificationContent");
+            return;
+        }
+
+        Notification fgNotification =
+                createNotification(channelId, channelName, androidNotificationTitle,
+                                   androidNotificationContent);
+        TempoServiceStatusListener tempoStatusListener = error -> {
+            if (error == null) {
+                onSuccess.invoke("Success");
```

Comment:
The string "Success" will be pass as a param in the `onSuccess` callback in JS? Do we need it?

Reply:
I have followed the pattern which we used with older API's, this can be removed as message is redundant

Reply:
fixed it